docs/software/communication/nccl.md

```diff
@@ -4,7 +4,43 @@
 [NCCL](https://developer.nvidia.com/nccl) is an optimized inter-GPU communication library for NVIDIA GPUs.
 It is commonly used in machine learning frameworks, but traditional scientific applications can also benefit from NCCL.
 
+## Using NCCL
+
+To use the Slingshot network on Alps, the [`aws-ofi-nccl`](https://github.com/aws/aws-ofi-nccl) plugin must be used.
+With the container engine, the [AWS OFI NCCL hook][ref-ce-aws-ofi-hook] can be used to load the plugin into the container and configure NCCL to use it.
+
+Most uenvs, like [`prgenv-gnu`][ref-uenv-prgenv-gnu], also contain the NCCL plugin.
+When using e.g. the `default` view of `prgenv-gnu` the `aws-ofi-nccl` plugin will be available in the environment.
+Alternatively, loading the `aws-ofi-nccl` module with the `modules` view also makes the plugin available in the environment.
+The environment variables described below must still be set to ensure that NCCL uses the plugin.
+
+While the container engine sets these automatically when using the NCCL hook, the following environment variables should always be set for correctness and optimal performance when using NCCL:
+
+```bash
+export NCCL_NET="AWS Libfabric" # (1)
```

Comment:
To do: document (not) using `NCCL_NET_PLUGIN=ofi` and the error message if both are used.

Reply:
https://github.com/eth-cscs/cscs-docs/pull/75/commits/4b7ae6b7df1dfdd3bfdbbf1c95c86e497aee26e4